helm/scality-cosi-driver/values.yaml

```diff
@@ -8,8 +8,8 @@ replicaCount: 1
 
 
 logLevels:
-  driver: "5"
-  sidecar: "5"
+  driver: "2"
```

Comment:
Could be nice to add comment with the text name of the log level.

Reply:
Thanks added in kustomize files and helm